includes/sanitizers/class-amp-object-sanitizer.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Class AMP_Object_Sanitizer
+ *
+ * @package AMP
+ */
+
+use AmpProject\Attribute;
+use AmpProject\Layout;
+
+/**
+ * Class AMP_Object_Sanitizer
+ *
+ * Sanitizes `<object>` embeds.
+ *
+ * @since 2.1
+ * @internal
+ */
```

Comment:
Just as a note for now: the file and class docblocks for the sanitizers need some general overhaul. I'm not marking anything here to fix this for this PR, as the above is consistent with the rest. But the current way of documenting these:

- Fails to take all WPCS rules into account.
- Has value-less content that just repeats the code.
- Has the wrong package annotation.